### ISSUE TITLE
Trained model and memory files

### ISSUE THREAD

**Issue by ashok-arora** [LiangZhang1996/AttentionLight#3]
Can you provide the trained model and the memory files used?

**Comment by LiangZhang1996** [comment]
The original trained model corresponding to the article results is about at the iniital of 2022, and it is hard to find.
If you really need these, I could re-train the model and provide it for you.

**Comment by ashok-arora** [comment]
Yes please, thank you very much. Would it be also possible to provide the RL memory file created in this process?
Also, is there a single intersection variant of it too?

**Comment by BackMountainDevil** [comment]
which trained model you need? or mem files? I train a lot. Quite fast
